Gruntfile.js

```diff
@@ -0,0 +1,23 @@
+module.exports = function(grunt) {
+
+    // Project configuration.
+    grunt.initConfig({
+      pkg: grunt.file.readJSON('package.json'),
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/victor0x16/miniature-adventure/pullRequest?prid=9070052)
```suggestion
      pkg: grunt.file.readJSON("package.json"),
```